EMIP/1-1000/EMIP00166.xml

```diff
@@ -172,6 +237,50 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                        </layout>
                      </layoutDesc>
                   </objectDesc>
+                  <handDesc>
+                    <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="rubrication">
+                        <foreign xml:lang="gez">ማርያም፡</foreign>
+                        <foreign xml:lang="gez">ሚካኤል፡</foreign>
+                         </seg>
+                    </handNote>
+                  </handDesc>
+
+                  <decoDesc>
+                      <decoNote type="miniature" xml:id="d01"><locus target="#iv(erso)"/><desc>The Madonna and Child, <ref type="authFile" corresp="AT1031VirginAndChild"></ref> with caption "Virgin Mary"</desc></decoNote>
+                      <decoNote type="miniature" xml:id="d02"><locus target="#iiv(erso)"/><desc>The Archangel Michael <ref type = "authFile" corresp ="AT1077ArchangelMichael"/></desc> with insufficiently legible caption</decoNote>
```

Comment:
note that this syntax  ref = " is wrong, no spaces should be used between the attribute name and the =" for its value.
it needs to be ref=".." 
any standard XML editor would prevent this behaviour

Reply:
This occurs in all files in this commit, please fix it everywhere

Reply:
Noted. Atom seems to allow this without any comment.